src/current/_includes/releases/v23.2/v23.2.0-alpha.4.md

```diff
@@ -0,0 +1,197 @@
+
+<h3 id="v23-2-0-alpha-4-db-console-changes">DB Console changes</h3>
+
+- The [**Jobs** table](../v23.2/ui-jobs-page.html) will now correctly display timestamps for creation, last modified, and the completed time fields. [#110366][#110366]
+- The [transaction insight details](../v23.2/ui-insights-page.html) will show the following details when CockroachDB has information on a transaction execution with a `40001` error code and it has captured the conflicting transaction meta details (only available if the transaction had not yet committed at the time of execution). A section called `Failed Execution` will appear when this information is available and it will contain:
+    - Blocking transaction execution ID
+    - Blocking transaction fingerprint ID
+    - Conflict location
+    - Database, table, and index names [#111873][#111873]
+- Added progressive loading functionality to the [Databases page](../v23.2/ui-databases-page.html). [#110901][#110901]
+
+<h3 id="v23-2-0-alpha-4-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in physical cluster replication where the primary cluster would not be able to take [backups](../v23.2/take-full-and-incremental-backups.html) when a primary cluster node was unavailable. [#111337][#111337]
+- Fixed a bug in [transaction insight details](../v23.2/ui-insights-page.html) where it was possible to see the contention details of other transactions. Now, CockroachDB will only surface contention details for the current transaction. [#111867][#111867]
+- [Voter constraints](../v23.2/configure-replication-zones.html) will now be satisfied by promoting existing non-voters. Previously, there was a bug where voter constraints were never satisfied due to all existing replicas being considered necessary to satisfy a replica constraint. [#111609][#111609]
+- Fixed a bug where `indoption` inside `pg_index` was not properly encoded causing clients to be unable to decode it as `int2vector`. [#111911][#111911]
+- This patch fixes an issue where the [optimizer](../v23.2/cost-based-optimizer.html) fails to honor the `statement_timeout` session setting when generating constrained index scans for queries with large `IN` lists or `= ANY` predicates on multiple index key columns, which may lead to an out of memory condition on the node. [#111979][#111979]
+- This patch fixes a performance issue in join queries with a `LIMIT` clause, where the [optimizer](../v23.2/cost-based-optimizer.html) may fail to push a `WHERE` clause filter into a join due to how the `LIMIT` operation is internally rewritten. This causes a full scan of the table referenced in the filter. [#110593][#110593]
+- Fixed a bug that caused internal errors during query optimization in rare cases. The bug has been present since version v2.1.11, but it is more likely to occur in version v21.2.0 and later, though it is still rare. The bug only presents when a query contains `min` and `max` aggregate functions. [#112255][#112255]
```

Comment:
link "aggregate functions" to https://www.cockroachlabs.com/docs/v23.2/functions-and-operators#aggregate-functions

Reply:
done